pkg/apis/core/validation/validation.go

```diff
@@ -5530,7 +5530,6 @@ func ValidateResourceQuantityValue(resource string, value resource.Quantity, fld
 // newResourceQuota is updated with fields that cannot be changed.
 func ValidateResourceQuotaUpdate(newResourceQuota, oldResourceQuota *core.ResourceQuota) field.ErrorList {
 	allErrs := ValidateObjectMetaUpdate(&newResourceQuota.ObjectMeta, &oldResourceQuota.ObjectMeta, field.NewPath("metadata"))
-	allErrs = append(allErrs, ValidateResourceQuotaSpec(&newResourceQuota.Spec, field.NewPath("spec"))...)
```

Comment:
ok, so the strategy calls both

ValidateResourceQuota
ValidateResourceQuotaUpdate

and both operations revalidate the spec.

I agree it seems fine to only validate the spec in ValidateResourceQuota, and keep the update operation focused on allowed updates.

Reply:
would update operation possibly update spec, hence spec needs to be validated again?